### PR TITLE
OSSMDOC-330: Extensions update

### PR DESCRIPTION
**Pull request by neal-timpe**
- Aligned team: Service mesh
- For branches: 4.6, 4.7, 4.8
- Changed two pieces from examples to procedures. Style updates. 
- https://issues.redhat.com/browse/OSSMDOC-330
- https://deploy-preview-33762--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-extensions?utm_source=github&utm_campaign=bot_dp
- SME review: knrc
- QE review: yxun
- Peer review

### CODE REVIEW

File: modules/ossm-extensions-wasm-deploy.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+kind: ServiceMeshExtension
+metadata:
+  name: header-append
+  namespace: istio-system
+spec:
+  workloadSelector:
+    labels:
+      app: httpbin
+  config: test
+  image: quay.io/maistra-dev/header-append-filter:latest
```
Comment (yxun) [inline modules/ossm-extensions-wasm-deploy.adoc:23]: @neal-timpe The line 23 "image: quay.io/maistra-dev/header-append-filter:latest" is an image for OSSM 2.1 release.
For current 2.0.x release, we need to use "image: quay.io/maistra-dev/header-append-filter:2.0" . Could you add a note or update that ? Thanks.

**Review by yxun — APPROVED** [review]

**Review by knrc — APPROVED** [review]

File: modules/ossm-extensions-wasm-support.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
+[id="ossm-extensions-support_{context}"]
+= Enabling WebAssembly extension support
+
+Support for WebAssembly extensions to {ProductName} is currently in Tech Preview, so it must be explicitly enabled for your `ServiceMeshControlPlane`.
```
Comment (bmcelvee) [inline modules/ossm-extensions-wasm-support.adoc:4]: Maybe add the link for technology preview:

Support for WebAssembly extensions to {ProductName} is currently in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview], so it must be explicitly enabled for your `ServiceMeshControlPlane`.

File: modules/ossm-extensions-wasm-deploy.adoc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  workloadSelector:
+    labels:
+      app: httpbin
+  config: test
+  image: quay.io/maistra-dev/header-append-filter:2.0
+  phase: PostAuthZ
+  priority: 100
+----
+
+. Apply the `extension.yaml` file with the following command.
```
Comment (bmcelvee) [inline modules/ossm-extensions-wasm-deploy.adoc:28]: s/command./command:

File: modules/ossm-extensions-wasm-deploy.adoc [hunk 4]
```diff
@@ -0,0 +1,33 @@
+[id="ossm-extensions-deploy_{context}"]
+= Deploying extensions
+
+{ProductName} extensions can be enabled using the `ServiceMeshExtension` resource. 
+
+.Procedure
+
+. Create the following example resource.
```
Comment (bmcelvee) [inline modules/ossm-extensions-wasm-deploy.adoc:8]: s/resource./resource:

**Comment by bmcelvee** [comment]
A few suggestions and a reminder to squash commits, otherwise LGTM!

**Comment by neal-timpe** [comment]
@bmcelvee Knocked those out too and just squashed. Ready for this one to merge and cp to 4.6-4.8 whenever you get another minute. Thanks for the review and merging the other one!

**Pull request merged**